scripts/dev/set_env_context.sh

```diff
@@ -8,12 +8,16 @@ source scripts/funcs/errors
 
 script_name=$(readlink -f "${BASH_SOURCE[0]}")
 script_dir=$(dirname "${script_name}")
-context_file="$(realpath "${script_dir}/../../.generated/context.export.env")"
 
-if [[ ! -f ${context_file} ]]; then
-    fatal "File ${context_file} not found! Make sure to follow this guide to get started: https://wiki.corp.mongodb.com/display/MMS/Setting+up+local+development+and+E2E+testing#SettinguplocaldevelopmentandE2Etesting-GettingStartedGuide(VariantSwitching)"
+tmpfile="$(mktemp)"
+if ! scripts/dev/regenerate_context.sh >"${tmpfile}" 2>&1; then
```

Comment:
What this line of code does?

Reply:
It dumps the output of the make switch performed under the hood to the file. Currently switch is a bit verbose (about 6lines of output). But we don't want to be silent if there is any problem in switching the context

Reply:
Will this work as well?

```shell
result=$(scripts/dev/regenerate_context.sh 2>&1)
exit=$?
if exit; then
echo result
fi
```